lib/util/cache/package/decorator.ts

```diff
@@ -119,7 +119,7 @@ export function cache<T>({
       await packageCache.set(finalNamespace, finalKey, result, ttlMinutes);
       return result;
     } catch (err) /* istanbul ignore next */ {
-      logger.error({ err }, 'cache decorate error');
+      logger.trace({ err }, 'cache decorate error');
```

Comment:
🤔 I think we can drop this catch block. It has no value for me as errors are logged later anyways.

@rarkins WDYT?

Reply:
I'll take that 👍 as agreement. I'll update this PR to just drop the whole try/catch block.

Reply:
Done! We may need to revisit this if we start to use the cache decorator outside of datasources.